lib/ansible/modules/identity/cyberark/cyberark_credential.py

```diff
@@ -0,0 +1,259 @@
+
+DOCUMENTATION = """
+module: cyberark_credential
+short_description: Module for retrieval of CyberArk vaulted credential using PAS Web Services SDK through the Central Credential Provider
+author:
+ - Edward Nunez (@enunez-cyberark)
+ - Cyberark Bizdev (@cyberark-bizdev)
+ - erasmix (@erasmix)
+ - James Stutes (@jimmyjamcabd)
+version_added: 2.8
+description:
+    - Creates a URI for retrieving a credential from the Cyberark Vault through the Privileged
+      Account Security Web Services SDK by requesting access to a specific object through an Application ID
+      It returns an Ansible fact called I(cyberarkcredential) as a JSON message with object information
+      that can be used by other modules. Every module can use this fact as C(cyberarkcredential) parameter.
+options:
+    api_base_url:
+        description:
+            - A string containing the base URL of the server hosting the Central Credential Provider
+    validate_certs:
+        type: bool
+        default: 'true'
+        description:
+            - If C(false), SSL certificate chain will not be validated.  This should only
+              set to C(true) if you have a root CA certificate installed on each node.
+    app_id:
+        description:
+            - A string containing the Application ID authorized for retrieving the credential.
+    query:
+        description:
+            - A string containing details of the object being queried
+              parameters
+                Safe=<safe name>
+                Folder=<folder name within safe>
+                Object=<object name>
+                UserName=<username of object>
+                Address=<address listed for object>
+                Database=<optional file category for database objects>
+                PolicyID=<platform id managing object>.
+        required: True
+    client_cert:
+        required: False
+        description:
+            - A string containing the file location and name of the client certificate used for authentication.
+    client_key:
+        required: False
+        description:
+            - A string containing the file location and name of the private key of the client certificate used for authentication.
+    reason:
+        required: False
+        description:
+            - Reason for requesting credential if required by policy.
+    state:
+        default: present
+        choices: [present]
+        description:
+            - Specifies the state.
+"""
+
+EXAMPLES = """
+- name: Retrieve credential from CyberArk Vault using PAS Web Services SDK via Central Credential Provider
+  cyberark_credential:
+    api_base_url: "{{ web_services_base_url }}"
+    app_id: "{{ application_id }}"
+    query: "Safe=test&UserName=admin"
+  register: cyberarkcredential
+  result:
+    '{ api_base_url }"/AIMWebService/api/Accounts?AppId="{ app_id }"&"{ query }'
+- name: Retrieve credential from CyberArk Vault using PAS Web Services SDK via Central Credential Provider
```

Comment:
For readability please include a blank line between tasks
```suggestion

- name: Retrieve credential from CyberArk Vault using PAS Web Services SDK via Central Credential Provider
```